Sources/SwiftCurrent_SwiftUI/Extensions/SwiftUILaunchStyleAdditions.swift

```diff
@@ -44,6 +51,7 @@ extension LaunchStyle {
             public var rawValue: LaunchStyle {
                 switch self {
                     case .navigationLink: return ._swiftUI_navigationLink
+                    case .modal: return ._swiftUI_modal
```

Comment:
We'll see how this PR ends up, but I'm wondering if it makes sense to make `.modal` function similarly as it does on UIKit where it takes in its own enumeration to style the modal further?  I haven't played with Modals/sheets in SwiftUI enough to figure out all the customizations we may be hiding with a single enumeration.

Reply:
I think we should do that in a follow-up PR. It should probably be the next thing we work on, though.